components/calendar/src/dangi.rs

```diff
@@ -711,6 +711,114 @@ mod test {
         }
 
         let cases = [
+            // TestCase { // This test case fails to match ICU
```

Comment:
nit: link to the issue you filed

suggestion: I think it's fine to uncomment these test cases with the correct values and leave a note saying that they don't match ICU with a link to the issue.

Reply:
Ok, I added the issue number with a comment for each test case that is inconsistent and adjusted the expected values so they pass.